docs/source/listings.rst

```diff
@@ -182,9 +182,19 @@ Pub/Sub Alert Streams
       -
 
     * - lsst-alerts
-      - LSST alert stream in Pub/Sub, cleaned of duplicate alerts.
+      - Avro serialized LSST alert stream in Pub/Sub, cleaned of duplicate alerts.
         Messages contain the original alert bytes and metadata.
 
+    * - lsst-alerts-json
+      - JSON serialized LSST alert stream in Pub/Sub, cleaned of duplicate alerts.
+        Messages contain the original alert bytes and metadata.
+
+    * - lsst-lite
+      - Lite version of lsst-alerts (every alert, subset of fields).
```

Comment:
```suggestion
      - Lite version of lsst-alerts-json (every alert, subset of fields).
```

.... right?

Reply:
@troyraen the `lsst-lite` Cloud Run service is triggered by the `lsst-alerts` Pub/Sub topic. Outgoing messages are JSON serialized, but incoming messages are not. I guess it is more appropriate to use `lsst-alerts-json` as you've suggested

Reply:
Oh, actually using `lsst-alerts` is fine. I can't think of any reason to prefer `lsst-alerts-json`.